sdk/src/main/java/tech/dojo/pay/sdk/card/data/entities/PaymentDetails.kt

```diff
@@ -5,7 +5,8 @@ import tech.dojo.pay.sdk.card.entities.DojoShippingDetails
 
 internal data class PaymentDetails(
     val cV2: String?,
-    val savePaymentMethod: Boolean? = null,
+    val savePaymentMethod: Boolean? = null, // this is for the SetUp Intent payments it needs to be true
```

Comment:
this comment doesn't make sense if we read it, it needs to be true, but is nullable and default it to null?